packages/devtools_app/lib/src/screens/memory/panes/diff/controller/diff_pane_controller.dart

```diff
@@ -29,8 +30,8 @@ class DiffPaneController extends DisposableController {
   final SnapshotTaker snapshotTaker;
 
   /// If true, a snapshot is being taken.
-  ValueListenable<bool> get isTakingSnapshot => _isTakingSnapshot;
-  final _isTakingSnapshot = ValueNotifier<bool>(false);
+  ValueListenable<bool> get isAddingSnapshot => _isAddingSnapshot;
```

Comment:
update dart doc to match renaming. Also add a comment to the dart doc that describes how a snapshot can be "added" (by taking a snapshot from a running dart application or by uploading from file)